ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js

```diff
@@ -186,4 +188,16 @@ describe('EditGasFeePopover', () => {
     });
     expect(screen.queryByText('10% increase')).toBeInTheDocument();
   });
+
+  it('should renders when advanced gas fee is set', () => {
+    render({
+      storeProps: { advancedGasFee: { maxBaseFee: 50, priorityFee: 2 } },
+      txProps: { userFeeLevel: 'custom', dappSuggestedGasFees: {} },
+    });
+
+    expect(screen.queryByText('⚙️')).toBeInTheDocument();
+    expect(screen.queryByText('Advanced')).toBeInTheDocument();
+    expect(screen.queryByTitle('0.0021 ETH')).toBeInTheDocument();
+    fireEvent.click(screen.queryByText('Advanced'));
+  });
```

Comment:
Here would be nice to have test to check that default advance fee is visible even if userFeeLevel is not custom.